asv.conf.json

```diff
@@ -4,20 +4,19 @@
     "project_url": "https://qiskit.org",
     "repo": ".",
     "install_command": [
-        "in-dir={env_dir} python -mpip install {wheel_file}[all] python-constraint qiskit-experiments==0.3.0"
+        "in-dir={env_dir} python -m pip install {wheel_file}[all] python-constraint qiskit-experiments==0.3.0"
       ],
     "uninstall_command": [
-        "return-code=any python -mpip uninstall -y {project}"
+        "return-code=any python -m pip uninstall -y qiskit qiskit-terra"
     ],
     "build_command": [
-        "pip install -U setuptools-rust",
-        "python setup.py build_rust --release",
-        "PIP_NO_BUILD_ISOLATION=false python -mpip wheel --no-deps --no-index -w {build_cache_dir} {build_dir}"
+        "python -m pip install -U build",
+        "python -m build --outdir {build_cache_dir} --wheel {build_dir}"
     ],
     "branches": ["main"],
     "dvcs": "git",
     "environment_type": "virtualenv",
-    "show_commit_url": "http://github.com/Qiskit/qiskit-terra/commit/",
+    "show_commit_url": "http://github.com/Qiskit/qiskit/commit/",
```

Comment:
This technically still worked because of the redirects :) but it's better to do this I agree

Reply:
Yeah for sure, and at the time that the asv benchmarks were merged in, the repo hadn't yet been renamed so it wasn't ready to be replaced then.